### PR TITLE
login via email and API for adding billing and shipping done

### PR DESCRIPTION
**Pull request by arohablue**

### CODE REVIEW

File: src/main/java/com/example/resellerapp/resellerapp/models/BillingProperties.java [hunk 1]
```diff
@@ -75,7 +75,7 @@
     private String email;
 
     @NotNull
-    @Size(max = 10)
+    //@Size(max = 10)
```
Comment (arohablue) [inline src/main/java/com/example/resellerapp/resellerapp/models/BillingProperties.java]: remove comment

File: src/main/java/com/example/resellerapp/resellerapp/models/User.java [hunk 2]
```diff
@@ -69,74 +72,26 @@
...
-    @Size(max = 10)
+    //@Size(max = 10)
     @Column(name = "phonenumber")
     private Integer phonenumber; //Customer first name.
 
     @OneToOne(fetch = FetchType.LAZY,
             cascade =  CascadeType.ALL,
             mappedBy = "user")
+        //@JsonIgnore
+        @JsonIgnoreProperties("user")
```
Comment (arohablue) [inline src/main/java/com/example/resellerapp/resellerapp/models/User.java]: when we fetch user details separately are we getting billing details or not?

File: src/main/java/com/example/resellerapp/resellerapp/models/User.java [hunk 3]
```diff
@@ -69,74 +72,26 @@
...
     private BillingProperties billingProperties;
 
     @OneToOne(fetch = FetchType.LAZY,
             cascade =  CascadeType.ALL,
             mappedBy = "user")
+        //@JsonIgnore
+        @JsonIgnoreProperties("user")
```
Comment (arohablue) [inline src/main/java/com/example/resellerapp/resellerapp/models/User.java]: when we fetch user details separately are we getting shipping details or not?

File: src/main/java/com/example/resellerapp/resellerapp/services/UserService.java [hunk 4]
```diff
@@ -40,6 +50,50 @@ public Boolean createUser(User user) {
...
+        public Boolean addShippingProperties(ShippingProperties shippingProperties) {
+            if (shippingRepository.save(shippingProperties) != null) {
+                return true;
+            }
+            return false;
+        }
+
+        @Override
+	    public BillingProperties getBillingByPhoneNumber(Integer phonenumber) {	
+            System.out.println(phonenumber);
```
Comment (arohablue) [inline src/main/java/com/example/resellerapp/resellerapp/services/UserService.java]: remove debug

File: src/main/java/com/example/resellerapp/resellerapp/services/UserService.java [hunk 5]
```diff
@@ -40,6 +50,50 @@ public Boolean createUser(User user) {
...
+                return billingProperties;
+                
+            } catch (Exception e) {
+                return null;                
+            }
+        }
+
+        @Override
+	    public ShippingProperties getShippingByPhoneNumber(Integer phonenumber) {	
+            System.out.println(phonenumber);
```
Comment (arohablue) [inline src/main/java/com/example/resellerapp/resellerapp/services/UserService.java]: remove debug.

File: src/main/java/com/example/resellerapp/resellerapp/models/User.java (hunk 2 above)
Comment (Tejas-Choudekar) [reply src/main/java/com/example/resellerapp/resellerapp/models/User.java]: We are getting

**Pull request merged**